### PR TITLE
Update links in README

### PR DESCRIPTION
**Pull request by vipulgupta2048**
More links in the README aren't working, is there a need to update them?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ npm i -D playwright
 
 This installs Playwright and browser binaries for Chromium, Firefox and WebKit. Once installed, you can `require` Playwright in a Node.js script and automate web browser interactions.
 
-* [Getting started](https://playwright.dev/#?path=docs/intro.md)
-* [Installation configuration](https://playwright.dev/#?path=docs/installation.md)
-* [API reference](https://playwright.dev/#?path=docs/api.md)
+* [Getting started](https://playwright.dev/docs/intro)
+* [Installation configuration](https://playwright.dev/docs/installation)
+* [API reference](https://playwright.dev/docs/api/)
```
Comment (mxschmitt) [inline README.md:27]:
```suggestion
* [API reference](https://playwright.dev/docs/api/playwright-module/)
```
I think this link fits better. Also not sure about trailing slashes or not
cc @arjunattam

Comment (vipulgupta2048) [reply README.md:27]: Thanks for the quick turnaround @mxschmitt I was looking around Playwright and found this non-responsive links. They are plenty more of them in the resources section of the readme

Comment (arjunattam) [reply README.md:27]: The trailing slash shouldn't make a difference. I believe the right link is this: 

```suggestion
* [API reference](https://playwright.dev/docs/api/class-playwright)
```

**Review by arjunattam — APPROVED** [review]
Thank you @vipulgupta2048

**Pull request merged**